packages/gatsby/src/services/initialize.ts

```diff
@@ -270,6 +269,7 @@ export async function initialize({
     ? lmdbCacheDirExists
     : cacheJsonDirExists
 
+  const { globby: glob } = await import(`globby`)
```

Comment:
What's the reason for this not to be a static import?

Reply:
It's ESM only, i'll add a comment
https://unpkg.com/globby@13.1.1/package.json